cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -906,13 +905,10 @@
         # Metadata
         head_node_launch_template.add_metadata("Comment", "AWS ParallelCluster Head Node")
         # CloudFormation::Init metadata
-        pre_install_action = head_node.get_custom_action(event=CustomActionEvent.NODE_START)
```

Comment:
Same comment. We are not deleting it, right? By the way, an integration test against it has already passed.

Reply:
Hanwen is right, let's restore CustomActions

Reply:
Done